cairo/src/font/font_options.rs

```diff
@@ -2,9 +2,9 @@
 
 #[cfg(feature = "v1_16")]
 use std::ffi::CString;
+use std::hash;
 #[cfg(not(feature = "use_glib"))]
 use std::ptr;
-use std::{cmp::PartialEq, hash};
```

Comment:
Why are you splitting this?

Reply:
i didn't split it, removed cmp::PartialEq as it is already in scope

Reply:
Oh and that then reordered `std::hash`, I see. Sorry :)